build.gradle.kts

```diff
@@ -267,7 +267,7 @@ subprojects {
         private val git = Grgit.open(mapOf("dir" to rootDir))
 
         override fun toString(): String {
-            return git.describe(mapOf("longDescr" to true, "tags" to true)) ?: git.head().abbreviatedId
+            return git.describe(mapOf("tags" to true)) ?: git.head().abbreviatedId
```

Comment:
Why not remove this commit if you are going to delete this line anyway in the next commit?

Reply:
Because I'm prepared to revert the next commit in case `reckon` turns out to be too inconvenient to use after all, and then this is the state I'd like to revert to.